v23.1/show-backup.md

```diff
@@ -158,7 +158,7 @@ system        | public             | role_members               | table       |
 To view an incremental backup that was taken with the `incremental_location` option, run `SHOW BACKUP` with the full backup and incremental backup location following the original `BACKUP` statement.
 
 {{site.data.alerts.callout_info}}
-`SHOW BACKUP` can display backups taken with the `incremental_location` option **or** for [locality-aware backups](take-and-restore-locality-aware-backups.html), but not for locality-aware backups taken with the `incremental_location` option.
+`SHOW BACKUP` can display metadata for locality-aware backups taken with the [`incremental_location`](show-backup.html#show-a-backup-taken-with-the-incremental-location-option) option.
```

Comment:
i'm not sure what the docs policy is here. Do we usually call out a fixed bug in the stable docs? I'll let you and the docs manager debate this.

Reply:
Since Michael mentions this was a bug fix, I don't think it is necessary to call out here. You can remove the note boxes with "can display" in v23.1 from both locations.

You're calling out in v22.2 that this wasn't possible, so that takes care of it. Plus there is a known limitation listed in v22.2. 

Yes! For v23.1, you can remove this known limitation, and therefore, the Known Limitation section from the bottom of the `SHOW BACKUP` page. (You will want to check that this is also removed from the [Known Limitations page](https://www.cockroachlabs.com/docs/dev/known-limitations.html#show-backup-does-not-work-with-locality-aware-backups-and-the-incremental_location-option).)

Reply:
Removed the "can display" boxes for v23.1 in the two examples. Also removed the known limitations sections below!